csrc/flash_attn_rocm/CMakeLists.txt

```diff
@@ -131,6 +131,9 @@ find_package(HIP)
 
 set(CMAKE_CXX_COMPILER /opt/rocm/hip/bin/hipcc)
 set(CMAKE_CXX_STANDARD 20)
+set (CMAKE_CXX_FLAGS "-ferror-limit=0")
+set (CMAKE_CXX_FLAGS "--offload-arch=gfx90a")
+set (CMAKE_CXX_FLAGS "-std=c++20")
```

Comment:
We have already set(CMAKE_CXX_STANDARD 20), there is not necessary to add "-std=c++20".